### PR TITLE
Play the state

### LINKED ISSUE #27: Don should read the states
#### Brief description of the issue:
Right now he doesn't/can't, because something broke.

#### Additional info:
 - **Bot Revision**: Pre-Release

### LINKED ISSUE #35: Audio clips should not be able to interrupt each other.
#### Brief description of the issue:
https://github.com/MTS11648/Don-NonAutomated-Bot/pull/34 cannot be merged because it is currently skipping the intro clip to immediately play the next clip. This is obviously not correct.

#### What happened?:
During the intro, bot will skip to "Are you currently living in (state)?"

#### What _should_ have happened?:
"Hi, this is Don from LCN (...) and are you currently living in (state)?"

#### Steps to reproduce:
- Play intro.
- ???
- Die a little inside.

#### Additional info:
 - **Bot Revision**: Pre-Release

### PR DESCRIPTION
**Pull request by SpadesNeil**
Resolves #35 
Resolves #27

### CODE REVIEW

File: Don-Manual-SoundBoard/Agent.cs [hunk 1]
```diff
@@ -1208,10 +1239,21 @@ private void PlayAutoFormQuestion()
...
+                        audioPath = @"C:\Soundboard\Don\STATES\New York\" + city + ".mp3";
+                        PlayAudioClip(audioPath);
+                    }
+                    else
+                    {
+                        audioPath = @"C:\Soundboard\Don\STATES\" + state + ".mp3";
+                        PlayAudioClip(audioPath);
+                    }
                     break;
                 case PROVIDER:
```
Comment (MTS11648) [inline Don-Manual-SoundBoard/Agent.cs]: don't forget that this needs to be changed with this addition

**Comment by SpadesNeil** [comment]
reeee

File: Don-Manual-SoundBoard/fileList.txt [hunk 2]
```diff
@@ -1,676 +1 @@
-﻿C:\SoundBoard\Don\Birthday\0101.mp3
```
Comment (SpadesNeil) [inline Don-Manual-SoundBoard/fileList.txt]: shit, remind me to put these back before merging this

**Comment by SpadesNeil** [comment]
a14

**Pull request merged**